src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -610,11 +621,14 @@ void InitSurfaceHeatBalance(EnergyPlusData &state)
             }
             // Store the calculated total zone Power Reduction Factor due to DElight daylighting
             // in the ZoneDaylight structure for later use
-            state.dataDaylightingData->ZoneDaylight(NZ).ZonePowerReductionFactor = dPowerReducFac;
+            thisDaylightControl.PowerReductionFactor = dPowerReducFac;
         }
         // RJH DElight Modification End - Call to DElight electric lighting control subroutine
     }
 
+    if (state.dataEnvrn->SunIsUp && !state.dataGlobal->DoingSizing) {
+        DayltgInteriorMapIllum(state);
+    }
```

Comment:
So as long as this will result in 0 for non-sun up hours

Reply:
Maps don't report anything for non-sun up hours, so OK.